cds/base/static/js/record_tools.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * This file is part of Invenio.
+ * Copyright (C) 2014 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * Invenio is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Invenio; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+define("js/record_tools",
+       [
+           "require",
+           "jquery",
+           "bootstrap",
+           "hgn!./templates/modal_list"
+       ],
+       function(require, $, bootstrap, modal_list) {
+   'use strict';
+
+    var tpl_list_modal = modal_list;
+
+    $("a[data-show-more]").each(function() {
+
+        // get json from data-items attribute
+        var raw_data_items = JSON.parse($(this).attr("data-items"));
+
+        var data_items = [];
+        raw_data_items.forEach(function(entry) {
+            data_items.push({'item': entry});
+        });
+
+        var modal_id = Date.now();
+
+        var modalContext = {
+            'modal_id': modal_id,
+            'list_items': data_items,
+            'title': $(this).attr("data-title"),
+            'close': $(this).attr("data-close-button")
+        };
+
+        $(this).attr("data-toggle", "modal");
+        $(this).attr("data-target", "#" + modal_id);
```

Comment:
I would trust jQuery's [`data()`](http://api.jquery.com/data/) function instead of [`attr`](http://api.jquery.com/attr/).

`data()` will also do the `JSON.parse()` (from above) automagically.

```diff
- var raw_data_items = JSON.parse($(this).attr("data-items");
+ var raw_data_items = $(this).data('items');
```